Mod Manager/src/routes/modList/+page.svelte

```diff
@@ -806,4 +959,36 @@
 	:global(.bx--snippet.bx--snippet--single) {
 		background-color: #262626;
 	}
+
+	.overflow-y-auto {
+		color-scheme: dark !important;
+	}
+
+	:global(.bx--inline-notification) {
+		margin-right: 10px !important;
+		margin-left: 8px !important;
+		background:#2F2F2F !important;
+		max-width: 100% !important;
+		margin-top: 0rem !important;
+		margin-bottom: 0rem !important;
+	}
+
+	:global(.bx--modal-content) {
+		overflow-y: visible !important;
+	}
+
+	@media (min-width: 42rem) {:global(.bx--modal-container) {
+		position: relative !important;
+		height: auto !important;
+		width: 90% !important;
+		min-height: 30rem !important;
+	}}
+
+	:global(.bx--modal-content) {
+		margin-bottom:1.8rem !important;
+	}
+
+	:global(.bx--modal-header) {
+		margin-bottom:0rem !important;
+	}
```

Comment:
Same thing with all these global styles for the modal classes. For the inline notification, it may be best to copy the final HTML elements and style those individually instead of using the component given how many changes are being made).